alf/bin/train.py

```diff
@@ -87,9 +85,17 @@ def train_eval(ml_type, root_dir):
 
 def main(_):
     FLAGS.alsologtostderr = True
+    random_seed = alf.get_config('TrainerConfig.random_seed')
+    alf.config(
+        'TrainerConfig',
+        random_seed=common.set_random_seed(random_seed),
+        raise_if_used=False)
     conf_file = common.get_conf_file()
-    common.parse_conf_file(conf_file)
-    train_eval(FLAGS.ml_type, FLAGS.root_dir)
+    try:
+        common.parse_conf_file(conf_file)
```

Comment:
will it raise error if parse_conf_file tries to set random seed again?

Reply:
moved all random_seed ralated code to get_env

Reply:
Looks nice!